frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -144,6 +146,39 @@ function ReaderCoptListener:updatePageInfoOverride(pageno)
     if self.reading_percent == 1 then
         page_info = page_info .. percentage_pre .. percentage_fmt:format(percentage*100) .. percentage_post
     end
+
+    if self.battery == 1 and self.battery_percent == 1 then -- append battery percentage
+        local batt_pre = "["
+        local batt_post = "]"
```

Comment:
Is it ok these `[ ]` are gone when the battery is charging?
I think they somehow provide the shape of a bettery icon/level `[ 45% ]` (while also making it different from the book reading progress just before it), it's strange they go away when the battery is charging.